src/20231207_AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnership/AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnership_20231207.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+
+/**
+ * @title TokenLogic & karpatkey Service Provider Partnership
+ * @author efecarranza.eth
+ * - Snapshot: TODO
+ * - Discussion: https://governance.aave.com/t/temp-check-financial-services-proposal-karpatkey-tokenlogic/15633
```

Comment:
Link should be to ARFC, to be consistent on for example the assets used (GHO on ARFC, vs others in TEMP CHECK)

Reply:
Done.